lang/swi-prolog/Portfile

```diff
@@ -71,12 +72,11 @@ configure.pre_args  \
 	-DCMAKE_INCLUDE_PATH=${prefix}/include      \
 	-DCMAKE_LIBRARY_PATH=/usr/lib:${prefix}/lib
 
-pre-build {
-	system "cd ${worksrcpath} && ./scripts/pgo-compile.sh"
-}
+test.run no
+test.cmd ctest --output-on-failure
```

Comment:
if you do not want to run tests (default), you don't need to set these two `test.*` fields.